Dockerfile

```
@@ -1,3 +1,44 @@
+## server build
+FROM openjdk:11.0.10 as serverbuild
+
+COPY app/server/ /app
+
+WORKDIR /app
+
+RUN apt-get update;apt-get install -y maven rsync; apt-get clean; rm -rf /var/lib/apt/lists/*
+
+RUN ./build.sh -DskipTests
+
+## rts build
+FROM node:14.15.4 as rtsbuild
+
+COPY app/rts /app
+
+WORKDIR /app
+
+RUN yarn install --frozen-lockfile
+
+RUN npx tsc
+
+## client build
+
+FROM node:14.15.4 as clientbuild
```

Looks like we've upgraded here. 😅

https://github.com/appsmithorg/appsmith/blob/73e43fadbd7e5aaa6d87e3419cf0106976ab92d3/app/client/package.json#L6

```suggestion
FROM node:16.14.0 as clientbuild
```